README.md

```diff
@@ -1,2 +1,85 @@
 # business-analytics
 Tools to produce Riff Business analytic reports on RiffEdu and RiffPlatform deployments
+
+This repository uses a third-party package called [Metabase][] to visualize usage metrics for Mattermost. Two docker containers are run; one using the metabase image and another using the
+official mysql image. Commands are provided to restore sql database backups from Mattermost to the mysql container.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Database Backups
+
+Download sql backups from the Riff Learning backup location (ask for the URL if you don't have it), and place all .sql files in the folder called 'database-backups' in the root of this repo.
+
+### Prerequisites
+
+- [docker CE][docker-install] (Community Edition)
+- [docker-compose][docker-compose-install]
+- git
+- make
+
+
+## Running the Software
+The first step is to clone this repository. The following commands are defined in a Makefile, and can be run as follows:
+
+```
+make up
```

Comment:
I am unable to run`make up`. I receive the output below. I do not appear to have any active docker containers. It looks like there's an error binding to the necessary ports:

`Starting Riff Business Analytics.
Creating network "business-analytics_default" with the default driver
Creating ba-mysql ... error

ERROR: for ba-mysql  Cannot start service ba-mmdb: driver failed programming external connectivity on endpoint ba-mysql (f568f539792f3a4beec3c188dd97ae70bed0f36cf3790e467635bff7762846c0): Error starting userland proxy: listen tcp 0.0.0.0:3306: bind: address already in use

ERROR: for ba-mmdb  Cannot start service ba-mmdb: driver failed programming external connectivity on endpoint ba-mysql (f568f539792f3a4beec3c188dd97ae70bed0f36cf3790e467635bff7762846c0): Error starting userland proxy: listen tcp 0.0.0.0:3306: bind: address already in use
ERROR: Encountered errors while bringing up the project.
Makefile:15: recipe for target 'up' failed
make: *** [up] Error 1`

Reply:
This error is caused because the ba-mysql server attempts to bind to the default port used by mysql. Since I already had a mysql server active locally, this caused the build to fail.

The docs should either instruct users to shutdown any active mysql instances (e.g. `#mysqld stop`), or the `docker-compose` should bind to a different port that is less likely to be in use.

Reply:
After stopping my local mysql instance, I was able to successfully run `make up`.

Reply:
Added this to the documentation under 'Getting Started'.